app/scripts/lib/constants.js

```diff
@@ -78,7 +78,12 @@ define([], function () {
     ACCESS_TYPE_ONLINE: 'online',
     ACCESS_TYPE_OFFLINE: 'offline',
 
-    DEFAULT_XHR_TIMEOUT_MS: 2500
+    DEFAULT_XHR_TIMEOUT_MS: 2500,
+
+    EVENTS: {
```

Comment:
constants.js feels like the wrong place for these events. For event names, unless they can change (like the messages used by the brokers to communicate with the browser), I like just inlining the messages directly. `signin.success` in code is every bit as good as `EVENTS.SIGNIN_SUCCESS`, sure, `signin.success` might change, but not *that* likely.